src/components/PollOptions/index.js

```diff
@@ -113,7 +113,7 @@ const PollOptions = ({
 const styles = StyleSheet.create({
   pollOptionsItemContainer: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the `backgroundColor` property in the `pollOptionsItemContainer` style seems fine. It's a simple color change and should not affect the functionality of the component.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->